app/components/TaskList.tsx

```diff
@@ -39,7 +127,7 @@ const TaskList: React.FC<Props> = ({ projectRevision }) => {
   }, [id, router]);
 
   return (
-    <div>
+    <div style={{ width: "500px" }}>
```

Comment:
We should be able to have less than 500px for the width. It should also be specified in the `<style>` element at the bottom, not here.